pkg/slack/slack_test.go

```diff
@@ -43,6 +43,24 @@ func TestBuildJobParams(t *testing.T) {
 			expected:    map[string]string{"KEY1": "VALUE1"},
 			errorString: "",
 		},
+		{
+			name:        "OneDEVSCRIPTS_CONFIGParameter",
+			params:      "\"DEVSCRIPTS_CONFIG=KEY1=VALUE1\"",
+			expected:    map[string]string{"DEVSCRIPTS_CONFIG": "KEY1=VALUE1"},
+			errorString: "",
+		},
+		{
+			name:        "TwoDEVSCRIPTS_CONFIGParameters",
+			params:      "\"DEVSCRIPTS_CONFIG=KEY1=VALUE1;KEY2=VALUE2\"",
+			expected:    map[string]string{"DEVSCRIPTS_CONFIG": "KEY1=VALUE1\nKEY2=VALUE2"},
+			errorString: "",
+		},
+		{
+			name:        "OtherParametersWithDEVSCRIPTS_CONFIGParameters",
+			params:      "\"BASELINE_CAPABILITY_SET=None\",\"DEVSCRIPTS_CONFIG=KEY1=VALUE1;KEY2=VALUE2\"",
+			expected:    map[string]string{"BASELINE_CAPABILITY_SET": "None", "DEVSCRIPTS_CONFIG": "KEY1=VALUE1\nKEY2=VALUE2"},
+			errorString: "",
+		},
```

Comment:
Update these to be generic to match the modified logic from above.  Prefer `Single`, `Multiple`, `Mixed` for naming structures instead of `One`, `Two`, etc.